unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/SectionsModule/SettingsSectionModel.cs

```diff
@@ -0,0 +1,40 @@
+using DCL.SettingsPanelHUD.Widgets;
+using ReorderableList;
+using UnityEngine;
+
+namespace DCL.SettingsPanelHUD.Sections
+{
+    [System.Serializable]
+    public class SettingsWidgetList : ReorderableArray<SettingsWidgetModel>
+    {
+    }
+
+    [CreateAssetMenu(menuName = "Settings/Configuration/Section", fileName = "SectionConfiguration")]
+    public class SettingsSectionModel : ScriptableObject
+    {
+        public Sprite icon;
+        public string text;
+        public SettingsButtonEntry menuButtonPrefab;
+        public SettingsSectionView sectionPrefab;
+        public SettingsSectionController sectionController;
+
+        [Reorderable]
+        public SettingsWidgetList widgets;
+
+        public SettingsSectionModel(
```

Comment:
Same comment about constructors in SOs as above.

Reply:
Fixed in this new PR (in order to avoid mix things between the part I and II): https://github.com/decentraland/explorer/pull/1784